src/Components/Base/BalanceListener.ts

```diff
@@ -0,0 +1,50 @@
+
+
+import { Subscription, interval, from, of } from 'rxjs'
+import { flatMap, switchMap } from 'rxjs/operators'
+const LISTENER_INTERVAL = 3000
+
+export default class BalanceListener {
+    private _subscriptions$: Subscription[] = []
```

Comment:
Array no longer needed, will be  a a `subscription: Subscription`

Reply:
done